### DISCUSSION TITLE
api call "MYMPD_API_PLAYER_OUTPUT_LIST" error

### DISCUSSION THREAD

**Discussion by payback007** [jcorporation/myMPD#598] [Q&A] [answered]
myMPD version: [8.1.2]

Hi, 

I'm trying to get the current output list by api call with following script:

```
#!/bin/sh

DATA=$(curl -s -k http://IP:PORT/api/ -d'{"jsonrpc":"2.0","id":0,"method":"MYMPD_API_PLAYER_OUTPUT_LIST","params":{}}')
echo "${DATA}"
```

Following error is reported:
```
{"jsonrpc":"2.0","id":0,"error":{"method":"MYMPD_API_PLAYER_OUTPUT_LIST","facility":"general","severity":"error","message":"JSON path \"$.params.partition\" not found or value is not string type, found type is \"MJSON_TOK_INVALID\"","data":{}}}
```

What I'm doing wrong?

**Comment by jcorporation** [comment]
This api method requires a partition parameter. 

For reference: https://jcorporation.github.io/myMPD/references/api/methods

**Reply by payback007** [reply]
Thank you very much for the fast feedback, but unfortunately I'm not so familiar with API calls. Can you please give an example? The API documentation I already found before, but I don't know how to proceed.

**Comment by payback007** [comment, marked as answer]
Found the solution, the correct call is:

```
#!/bin/sh

DATA=$(curl -s -k http://IP:PORT/api/ -d'{"jsonrpc":"2.0","id":0,"method":"MYMPD_API_PLAYER_OUTPUT_LIST","params":{"partition":""}}')
echo "${DATA}"
```
